cfgov/data_research/handlers.py

```diff
@@ -31,10 +31,14 @@ def process(self, is_submitted):
         return {'form': ConferenceRegistrationForm()}
 
     def is_at_capacity(self):
-        query = reduce(
-            lambda x, y: Q(('codes__contains', x)) & Q(('codes__contains', y)),
-            self.block_value['codes']
-        )
+        query = Q()
+
+        # TODO: Let's perhaps use a less-dangerous way to compare
+        # codes than a straight string-compare against a JSON list
+        # Alternatively (less-safe), make sure codes are strictly
+        # very-unique and not subsets of each other.
+        for code in self.block_value['codes']:
+            query &= Q(codes__contains=code)
```

Comment:
I'm curious to hear more about this... I don't know that I'm fully aware of what the return JSON looks like.

I'd recommend sticking something on either our or data & research's backlog.